### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by hugovk**
| Q             | A
| ------------- | ---
| Bug fix?      | yes 
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no
| License       | MIT

Fix some typos.

### CODE REVIEW

File: src/Util/CloneTrait.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 namespace ApiPlatform\Core\Util;
 
 /**
- * Clones given data if cloneable.
+ * Clones given data if clonable.
```
Comment (teohhanhui) [inline src/Util/CloneTrait.php]: "Cloneable" is correct: https://www.php.net/manual/en/reflectionclass.iscloneable.php

Please revert.

**Comment by teohhanhui** [comment]
Also, please cherry-pick this for the `2.5` branch.

File: src/Util/CloneTrait.php (hunk 1 above)
Comment (hugovk) [reply src/Util/CloneTrait.php]: Thanks, reverted!

**Comment by hugovk** [comment]
> Also, please cherry-pick this for the `2.5` branch.

Please see PR https://github.com/api-platform/core/pull/3210.

**Pull request closed without merging**